.eslintrc.js

```diff
@@ -40,5 +29,18 @@ module.exports = {
     'padded-blocks': process.env.NODE_ENV  === 'development' ? 'warn' : 'error',
     'keyword-spacing': process.env.NODE_ENV  === 'development' ? 'warn' : 'error',
     'arrow-spacing': process.env.NODE_ENV  === 'development' ? 'warn' : 'error'
-  }
+  },
+  parserOptions: {
+    parser: 'babel-eslint',
+  },
+  overrides: [
+    {
+      files: [
+        '**/__tests__/*.{j,t}s?(x)'
+      ],
+      env: {
+        mocha: true
+      }
+    }
+  ]
```

Comment:
What is the purpose of this block? At least for the Karma tests, I didn`t have any trouble running or linting them without it.

Reply:
This block gives an extra-context to the files under the __tests__ directories. That way, the global variables which Mocha automatically adds (so you can use describe, it, etc... without importing anything in your spec files) are also known by ESLint. Otherwise it could potentially complain that some functions are used but not defined depending how you wrote your spec files.

`**/__tests__/*.{j,t}s?(x)` is not used in Wegue but this line is present inside the vue-cli template.

I also see that in the following commits (adding vue-cli4) which I haven't published yet, I also added `**/tests/unit/**/*.spec.{j,t}s?(x)` which IS what is needed to avoid complaints about undefined functions in Wegue... something I missed in the current PR sorry...

Reply:
Thanks for the clarification. The wildcard path to `__tests__` looked a bit strange to my eye. If this is in the default template, then I don`t have any problem keeping it in either way.

Reply:
In fact, this `__tests__` thing is one of the two recommended ways of structuring unit tests inside a `Vue` project. It is recommended to either put all the tests inside a `tests` directory (as done in `Wegue`) or to create multiple `__tests__` directories next to the files to which those tests relate.

[4a3b2a2](https://github.com/wegue-oss/wegue/pull/297/commits/4a3b2a29f4349675512ecdf69e1c4b4f62b47e7c) adds `**/tests/unit/**/*.spec.{j,t}s?(x)` as explained up here.